### ISSUE TITLE
Help Creating a Custom Site

### ISSUE THREAD

**Issue by Nash10se** [dobbelina/repository.dobbelina#653]
I tried to create a custom site for "Erome.com."

I downloaded the "crazyshit.com" zip file, and then went through all 4 files and replaced every mention of "crazyshit" or "Crazyshit" with "erome" or "Erome," and edited the filenames and such. Trid to install it from inside Cumination, and got a "installation cancelled or invalid file" error.

So if simply replacing "crazyshit" throughout the py file isn't sufficient, I have no idea how to create a custom channel. Any additional clues?

**Comment by Gujal00** [comment]
just replacing the name is a start. Now you have to replace the regex patterns to suit the new site

**Comment by ErosVece** [comment]
Is there any interest for a custom site tutorial? I'll pick eroma as an example.

**Comment by imurreflection** [comment]
Yes please..I am extremely interested..please provide detailed tutorial

**Comment by Nash10se** [comment]
I would be very interested as well. Thanks!

**Comment by imurreflection** [comment]
@erosvece...can you please start this as after learning,we might be able to provide some support on ongoing or future issues.

**Comment by dirtman69** [comment]
@ErosVece I'd be interested in tutorial

**Comment by ErosVece** [comment]
I'm currently very busy.

But step 1 would have been, learn the basics of Python. This is a good course: https://www.sololearn.com/learning/1073  (or https://www.sololearn.com/learning/1169)
Especially the 'Regular expressions' part is important.

So start there in advance :)

**Comment by ErosVece** [comment]
So, did everybody learn Python yet?

Still very busy with a new job (and the old job) :)

**Comment by Nash10se** [comment]
Congratulations on the new job! TBH, not likely to learn a whole coding language in my spare time just so a good porn site can work on my kodi install. It's super aspirational for me, sure. Just like losing 20 pounds.

**Comment by solo4201** [comment]
I did....

**Comment by solo4201** [comment]
I did....

**Comment by solo4201** [comment]
I learned that he really likes Cumination. I'm getting depressed we don't even go outside anymore. I also learned a bit of Python 3 In the few months that he's forced me to sit at the computer desk 24/7. I keep telling him now that I can speak python we might be able to spit some code at some real chicks, but he just wants me to add a bunch of custom sites. 
   I might have to figure this out on my own soon. I'm afraid he might go boa constrictor on me. I hope you get some time off that job soon, He doesn't trust me anymore and I think he knows I'm trying to escape.
  Sorry, I couldn't help myself.... :)
